src/main/kotlin/gg/essential/essential-loader.gradle.kts

```diff
@@ -0,0 +1,64 @@
+package gg.essential
+
+import gg.essential.gradle.multiversion.Platform
+import gg.essential.gradle.util.RelocationTransform.Companion.registerRelocationAttribute
+
+
+plugins {
+    id("gg.essential.loom")
+}
+
+val platform = Platform.of(project)
+
+val essentialLoader: Configuration by configurations.creating
+
+when {
+    platform.isLegacyForge -> {
+        dependencies {
+            "runtimeOnly"(essentialLoader("gg.essential:loader-launchwrapper:1.1.3")!!)
```

Comment:
Would it not make sense to provide a property override for the luanchwrapper version?

Reply:
The stage0 loader should never change. If it does, then yes I could see a need but the last time a stage0 loader changed was over a year ago.